std/traits.d

```diff
@@ -6967,3 +6970,118 @@ unittest
     struct BoolWrapper { bool value; }
     static assert(!ifTestable!(bool, a => BoolWrapper(a)));
 }
+
+/**
+ * Detect whether `X` is a type. Analogous to `is(X)`. This is useful when used
+ * in conjunction with other templates, e.g. `allSatisfy!(isType, X)`.
+ *
+ * Returns:
+ *     `bool`
```

Comment:
Returns: bool

that is just to little at least something like return true if X... is a type, false otherwise. This stands for all the other traits in this PR as well.